tests/src/debug.ts

```diff
@@ -0,0 +1,693 @@
+/**
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+
+
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+
+ * SPDX-License-Identifier: EPL-2.0
+
+ * Contributors:
+ *  Broadcom, Inc. - initial API and implementation
+ */
+
+/* eslint-disable @typescript-eslint/no-namespace */
+
+/// <reference types="cypress" />
+
+import { USER, PASS, CICSAPPLID, INTERTESTPORT, CONVJCL, ORIGJCL } from './cypressEnv';
+
+//@ts-ignore
+const rawTypeString = (str) => str.replace(/{/g, '{{}'); // Types the literal { key
+
+declare namespace Cypress {
+  interface Chainable<Subject> {
+    /**
+     * Read and write JSON file
+     * Copy from test_files/project/dap/ specific launch json file
+     * and paste to .theia/launch.json
+     * @example cy.readAndWriteJsonFile('TEST')
+     */
+    readAndWriteJsonFile(expression: string): Chainable<any>;
+  }
+}
+Cypress.Commands.add('readAndWriteJsonFile', (expression: string) => {
+  cy.readFile(`test_files/project/dap/${expression}.launch.json`).then((content) => {
```

Comment:
You have hardcoded `test_files` folder here, but it not appears in the PR. 
Will it be a separate change?

Reply:
`test_files` folder will appear in the test suite project (i.e. in COBOL LS Functional Test Suite), but it's in dev stage at the moment.